src/System.Linq/src/System/Linq/DebugView.cs

```diff
@@ -12,7 +11,10 @@ namespace System.Linq
     // NOTE: DO NOT DELETE THE FOLLOWING DEBUG VIEW TYPES.
     // Although it might be tempting due to them not be referenced anywhere in this library,
     // Visual Studio currently depends on their existence to enable the "Results" view in 
-    // watch windows.
```

Comment:
I'm thinking of the scenario where Visual Studio stops using them; they should perhaps still remain, or at least whether other users exist be investigated, rather than immediately deleted as cruft.